flow.c

```diff
@@ -7,7 +7,8 @@ struct fio_flow {
 	unsigned int refs;
 	struct flist_head list;
 	unsigned int id;
-	long long int flow_counter;
+	volatile unsigned long long flow_counter;
```

Comment:
Why is this volatile? Are you using it from a signal handler?

Reply:
the variable is incremented from different jobs, i wanted to get the value from memory instead of cache to increase the accuracy of the flow.   do you think i should fetch it using atomic api instead ?